robotoff/prediction/ocr/category_from_AOC.py

```diff
@@ -0,0 +1,84 @@
+import re
+from typing import Dict, Iterable, List, Optional, Union
+
+from robotoff import settings
+from robotoff.prediction.types import Prediction, PredictionType
+from robotoff.taxonomy import Taxonomy, TaxonomyNode, get_taxonomy
+from robotoff.utils import get_logger
+
+from .dataclass import OCRField, OCRRegex, OCRResult, get_text
+
+logger = get_logger(__name__)
+
+
+def process_category(lang, match) -> Optional[str]:
+    """Function to process the name of the category extracted from the AOC
+    recognition. We want it to match categories found in the categories
+    taxonomy database. If nothing is found, we don't return the category."""
+
+    unchecked_category = str(lang) + ":" + match.group().replace(" ", "-")
```

Comment:
If you use a named capturing expression (see comment about avoiding positive/negative  look behind, you can here use `match.group("category")`